packages/palette/src/themes/Themes.story.tsx

```diff
@@ -0,0 +1,224 @@
+import React from "react"
+import {
+  Box,
+  Column,
+  GridColumns,
+  Join,
+  Separator,
+  Spacer,
+  Text,
+  TextVariant,
+} from "../elements"
+import { useTheme } from "../Theme"
+
+export default {
+  title: "Theme",
+}
+
+export const Theme = () => {
+  const { theme } = useTheme()
+
+  return (
+    <>
+      <Text variant="xxl" mb={4}>
+        Theme
+      </Text>
+
+      <Box as="pre" my={0}>
+        {JSON.stringify(theme, null, 2)}
+      </Box>
+    </>
+  )
+}
+
+export const Colors = () => {
+  const { theme } = useTheme()
+  const colors = Object.entries(theme.colors)
+
+  return (
+    <Box>
+      <Text variant="xxl" mb={4}>
+        Color Palette
+      </Text>
+
+      <Text variant="sm">
+        The Artsy color palette has been updated to improve the accessibility.
+        Each color has been adjusted to ensure sufficient contrast and has
+        specific roles as to which colors can be used in conjunction with each
+        other.
+      </Text>
+
+      <Separator color="black30" my={12} />
+
+      <GridColumns>
+        <Column span={6}>
+          <Text variant="sm" color="black60">
+            Color Value
+          </Text>
+        </Column>
+
+        <Column span={6}>
+          <Text variant="sm" color="black60">
+            Hex Value
+          </Text>
+        </Column>
+      </GridColumns>
+
+      <Separator color="black30" my={6} />
+
+      {colors.map(([name, value]) => {
+        return (
+          <GridColumns key={name} justifyContent="space-between" my={4}>
+            <Column span={6} display="flex" alignItems="center">
+              <Box width={60} height={60} bg={name} borderRadius="50%" />
```

Comment:
do we not have a 6 spacing scale?

Reply:
width/height doesn't use the spacing scale

Reply:
oh duh